pkg/services/ngalert/notifier/channels/email.go

```diff
@@ -123,34 +140,13 @@ func (en *EmailNotifier) Notify(ctx context.Context, as ...*types.Alert) (bool,
 				"RuleUrl":           ruleURL,
 				"AlertPageUrl":      alertPageURL,
 			},
-			To:          en.Addresses,
-			SingleEmail: en.SingleEmail,
-			Template:    "ng_alert_notification",
+			EmbeddedFiles: embeddedFiles,
+			To:            en.Addresses,
+			SingleEmail:   en.SingleEmail,
+			Template:      "ng_alert_notification",
 		},
 	}
 
-	// TODO: modify the email sender code to support multiple file or image URL
-	// fields. We cannot use images from every alert yet.
-	_ = withStoredImage(ctx, en.log, en.images,
-		func(index int, image *ngmodels.Image) error {
-			if image == nil {
-				return nil
-			}
-
-			if len(image.URL) != 0 {
-				cmd.Data["ImageLink"] = image.URL
-			} else if len(image.Path) != 0 {
-				file, err := os.Stat(image.Path)
```

Comment:
I should go back to doing the os.Stat thing here instead of splitting the path and waiting for the email code to fail to open the file.